server/src/routes/api/v1/votesRouter.js

```diff
@@ -0,0 +1,58 @@
+import express from 'express'
+import { Vote } from '../../../models/index.js'
+import VoteSerializer from '../../../../serializers/VoteSerializer.js'
+import { Review } from '../../../models/index.js'
+
+const votesRouter = new express.Router()
+
+votesRouter.get('/:reviewId', async (req, res) => {
+  try {
+    const { reviewId } = req.params
+    const totalCount = await VoteSerializer.getTotal(reviewId)
+    return res.status(201).json({ totalCount })
+  } catch (error) {
+    console.log(error)
+    return res.status(500).json({ errors: error })
+  }
+})
+
+votesRouter.post('/:reviewId', async (req, res) => {
```

Comment:
A review has many votes, our RESTful path should instead be reversed here,
`/api/v1/reviews/:reviewId/votes`

Otherwise a path of `api/v1/votes/:someId` would indicate that the `id` in the path is for a specific vote, even if we name it `reviewId`